packages/core/src/modules/proofs/formats/presentation-exchange/PresentationExchangeProofFormat.ts

```diff
@@ -0,0 +1,40 @@
+import type { PresentationDefinition } from '../../../presentation-exchange'
+import type { W3cCredentialRecord } from '../../../vc'
+import type { ProofFormat } from '../ProofFormat'
+
+export interface PresentationExchangeProofFormat extends ProofFormat {
+  formatKey: 'presentationExchange'
+
+  proofFormats: {
+    createProposal: {
+      presentationDefinition: PresentationDefinition
+    }
+
+    acceptProposal: {
+      name?: string
+      version?: string
+    }
+
+    createRequest: {
+      presentationDefinition: PresentationDefinition
+    }
+
+    acceptRequest: never
+
+    getCredentialsForRequest: {
+      input: never
+      output: Array<W3cCredentialRecord>
+    }
+
+    selectCredentialsForRequest: {
+      input: never
+      output: Array<W3cCredentialRecord>
+    }
```

Comment:
Can we update these interfaces to align with the more advanced structure we have implemented in the openid4vp and pex services? We have quite an advanced structure to make it easier to detect which groups of credentials you can use to satisfy a request. Especially `getCredentialsForRequest` should not just return a w3c record array.

In addition, I think it can help to return an object with `credentials` property even for select method so if we need to add a property in the future it won't be a breaking change